### PR TITLE
update wisemodel url, add  FQA in loss evaluation

### PR DESCRIPTION
**Pull request by zhao1iang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,24 @@ loss = -\sum^{n}_{i=1} log(p_i) / n = -log( \prod_{i=1}^n p_i) / n
...
+
+**A2**: 根据问题1我们知道Llama模型一般是将汉字切分成3个字符，为了保证一篇文档输入模型分词后不超过4096的限制，所以我们将文档最大长度设置成了4096/3=1228。在我们对比模型中，Llama模型是对中文切分最细的，所以只要不超过Llama模型的分词长度，其他模型也肯定可以塞得下。 
+
+**Q3**: 不同模型有不同的最大长度，统一用4096是否不公平？
+
+**A3**: 如上所示，我们计算的文档长度是1228个中文汉字，以Qwen为例，训练长度为2K，推理时候可以扩展到8K，并且中英双语模型的压缩率一般在2-3倍，因此1228个中文字符一般只有500-1000个token，远远达到不了2K甚至4K的最大长度限制。 
+
+**Q4**: 为什么Average Ppl和每个领域Ppl平均不一致？
+
+**A4**: 我们计算Average Ppl的方式是将所有文档的loss平均起来，然后取指数转换成Ppl，这样是为了避免有些领域Ppl过分的大，导致整体结果不好看。物理含义是将所有文档看成是一个统一的问题，Average Ppl是在这个文档整体生的Ppl。
```
Comment (BBuf) [inline README.md:244]: 可以refine一下最后这句话，refine为：其物理含义是将所有文档视为一个整体问题，Average Ppl则是针对这整体文档计算出的Ppl。

**Review by BBuf — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zhao1iang) [reply README.md]: 已修改
